src/ssl.c

```diff
@@ -12036,6 +12036,172 @@ static int wolfSSL_remove_ciphers(char* list, int sz, const char* toRemove)
+            return NULL;
+        
+        XMEMSET(locallist, 0, len);
+        
+        head = locallist;
+        
+        if (!tls13Only)
+        {
+            /* always tls13 suites in the head position */
+            XSTRNCPY(locallist, list, len);
+            locallist += listsz;
+            *locallist++ = ':';
+            *locallist = 0;
+            len -= listsz + 1;
+        }
+        
+        for(idx = 0; idx < suites->suiteSz; idx++) {
+            cipherSuite0 = suites->suites[idx];
+            cipherSuite  = suites->suites[++idx];
+            
+            if (tls13Only && cipherSuite0 == TLS13_BYTE) {
+                enabledcs = GetCipherNameInternal(cipherSuite0, cipherSuite);
+            }
+            else if (!tls13Only && cipherSuite0 != TLS13_BYTE) {
+                enabledcs = GetCipherNameInternal(cipherSuite0, cipherSuite);
+            } 
+            else
+                continue;
+            
+            ianasz = (int)XSTRLEN(enabledcs);
+            if (ianasz + 1 < len) {
+                XSTRNCPY(locallist, enabledcs, len);
+                locallist += ianasz;
+                
+                *locallist++ = ':';
+                *locallist = 0;
+                len -= ianasz + 1;
+            }
+            else{
+                XFREE(locallist, ctx->heap, DYNAMIC_TYPE_TMP_BUFFER);
+                return NULL;
+            }
+        }
+        
+        if (tls13Only) {
+            XSTRNCPY(locallist, list, len);
+            locallist += listsz;
+            *locallist = 0;
+        }
+        
+        return head;
+    } 
+    else 
+        return NULL;
+}
+
+/*
+ * check if the list has TLS13 and pre-TLS13 suites
+ * @param list cipher suite list that user want to set
+ * @return mixed: 0, only pre-TLS13: 1, only TLS13: 2
+ */
+static int CheckcipherList(const char* list)
+{
+    int ret;
+    int findTLSv13Suites = 0;
+    int findbeforeSuites = 0;
+    byte cipherSuite0;
+    byte cipherSuite1;
+    int flags;
+    char* next = (char*)list;
+    
+    do {
+        char*  current = next;
+        char   name[MAX_SUITE_NAME + 1];
+        word32 length;
+        
+        next   = XSTRSTR(next, ":");
+        length = min(sizeof(name), !next ? (word32)XSTRLEN(current) /* last */
+                                         : (word32)(next - current));
+        XSTRNCPY(name, current, length);
+        name[(length == sizeof(name)) ? length - 1 : length] = 0;
+        
+        ret = wolfSSL_get_cipher_suite_from_name(name, &cipherSuite0, 
+                                                        &cipherSuite1, &flags);
+        if (ret == 0) {
+            if (cipherSuite0 == TLS13_BYTE) {
+                /* TLSv13 suite */
+                findTLSv13Suites = 1;
+                break;
+            } 
+            else {
+                findbeforeSuites = 1;
+                break;
+            }
+        } 
+        if (findTLSv13Suites == 1 && findbeforeSuites == 1)
```

Comment:
Put a braces around lines in if.
(Less of a maintenance issue.)

Reply:
Put braces